section4/exercises/classMethods.js

```diff
@@ -28,13 +28,30 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
 
-  // ADD CODE
+changeProtein() {
+  this.protein = "chicken";
```

Comment:
This method changes your protein to 'chicken' every time. How can you leverage a param to make this method dynamic?